### PR TITLE
Adding main page template for Sam

### PR DESCRIPTION
**Pull request by CarlosHL**
- Creating a frontend test directory with a sample.
- Testing html Bass library.

### CODE REVIEW

File: frontend test/htmlbass.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!-- index.html -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="http://d2v52k3cl9vedd.cloudfront.net/basscss/7.0.4/basscss.min.css" rel="stylesheet">
```
Comment (antsankov) [inline frontend test/htmlbass.html]: Will we be hosting this on the server, or via a CDN?

**Comment by antsankov** [comment]
+1 Looks good.

**Comment by CarlosHL** [comment]
Adding a mockup template

**Comment by Brady76** [comment]
I like the mock up!

**Comment by untra** [comment]
Lookin good. Will remove one unnecesarry file. :shipit:

**Pull request merged**